app/src/main/java/uq/ecosoft/ctrack/MainActivity.java

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
Will get re-factored. Can you make an issue, link it here before closing this review comment